internal/pkg/service/common/prefixtree/prefixtree_test.go

```diff
@@ -39,7 +39,8 @@ func TestPrefixTree(t *testing.T) {
 	assert.True(t, found)
 	assert.Equal(t, value{field: "value2"}, v)
 
-	// AllFromPrefix / FirstFromPrefix / LastFromPrefix - 2 items
+	// All/ AllFromPrefix / FirstFromPrefix / LastFromPrefix - 2 items
+	assert.Len(t, tree.All(), 2)
```

Comment:
Akorát ten WalkAll netestuješ, on sa sice volá interně v tom All, ale je public tak by měl mět taky assert.

Reply:
jj to sa tyka vsetkych `All/ AllFromPrefix / FirstFromPrefix / LastFromPrefix`, ono test pre `WalkAll` bude kopia `All` metody, doplnim.

Reply:
doplnene doplnene https://github.com/keboola/keboola-as-code/pull/1385/commits/7afd2e1bcc9675fd30c45bdd3a9ed2d9b0f5caa8